support-frontend/assets/components/paypalExpressButton/PayPalExpressButton.jsx

```diff
@@ -20,13 +20,9 @@ type PropTypes = {|
   onClick: Function,
   formClassName: string,
   isTestUser: boolean,
-  contributionType: ContributionType,
-  setupRecurringPayPalPayment: (
-    resolve: string => void,
-    reject: Error => void,
-    IsoCurrency, CsrfState,
-    regularContributionType: ContributionType
-  ) => void,
+  amount: number,
```

Comment:
Rather than passing the contribution type into this component so that it can work out the amount to charge and the billing period we now pass these in explicitly.

Reply:
how do we determine the currency?

Reply:
It's a prop, github just isn't showing it.